website/content/en/docs/deployment/pipeline-profiles.md

```diff
@@ -0,0 +1,122 @@
++++
+title = "Pipeline Profiles"
+description = "Use AWS IAM roles for service accounts with Kubeflow Profiles"
+weight = 70
++++
+
+### Pipeline S3 Credential Option IRSA (IAM roles for service accounts)
+When using IRSA (IAM roles for service accounts) as your pipeline S3 credential option any additional profiles created as part of a multi-user deployment besides the preconfigured `kubeflow-user-example-com` will need to be configured with IRSA to access S3. As part of multi-user-isolation, every Pipeline run in Kubeflow Pipelines is ran in an individuals profile user-namespace. A `default-editor` service account exists in every Profile's namespace that is used to run Pipelines as-well as connect to S3. This `default-editor` SA needs to be annotated with an IAM role with sufficient permissions to access your S3 Bucket to run your Pipelines. In the below steps we will be configuring an IAM role with restricted access to a specific S3 Bucket, we recommend following the principle of least privileges. 
```

Comment:
- will need to be configured with IRSA to access S3 -> will need to be configured with permissions to S3 bucket using IRSA.
- As part of multi-user-isolation -> In a multi tenant Kubeflow installation, 
- Pipeline run -> pipeline
- drop as-well as connect to S3

Maybe:

In a multi tenant Kubeflow installation, the pods created by pipelines workflow and the pipelines frontend services run in user profile namespace. The service account(`default-editor`) used for these pods needs permissions for the S3 bucket used by pipelines to read and write artifacts from S3. When using IRSA (IAM roles for service accounts) as your pipeline S3 credential option, any additional profiles created as part of a multi-user deployment besides the preconfigured `kubeflow-user-example-com` will need to be configured with permissions to S3 bucket using IRSA.

The `default-editor` SA needs to be annotated with an IAM role with sufficient permissions to access your S3 Bucket to run your pipelines. In the below steps we will be configuring a profile an IAM role with restricted access to a specific S3 Bucket using the `AwsIamForServiceAccount` plugin for Profiles. To learn more about the read the [Profiles component guide](https://awslabs.github.io/kubeflow-manifests/docs/component-guides/profiles/#3-create-a-profile)

Rough draft feel free to modify